pom.xml

```diff
@@ -86,6 +86,7 @@
     <odfdom-java.version>0.9.0</odfdom-java.version> <!-- do not update to 0.10.0, see issue #4397 -->
     <vicino.version>1.2</vicino.version>
     <clojure.version>1.11.1</clojure.version>
+    <httpclient.version>4.5.13</httpclient.version>
```

Comment:
Ah, this likely indicates that we also pull the version 4 in other modules, which is perhaps why the extensions did not require any adaptation: they probably can still use the version 4 through indirect dependencies.
It is worth using `mvn dependency:analyze` to make sure we only depend on one version of this library throughout the application.

Reply:
Sure, am looking into it